hugo/content/research/2024/ai-preview/index.md

```diff
@@ -27,34 +28,36 @@ This preview will close with some insights into the [impact of AI](#impact-of-ai
 
 ### Reliance: a shift in the development landscape
 
-We examined the extent to which developers rely on AI for various tasks related to software development. The results revealed a significant increase in AI adoption across a range of activities compared to last year's data.
+This year's report examines the extent to which developers rely on AI for various tasks related to software development. Comparing this year's results to last year's demonstrates that AI adoption has — and this might not be surprising to you — continued on a steady rise. Last year's question, however, is different from this year's question, making a direct comparison difficult.
```

Comment:
Should this be "Last year's question**s** **are** different from this year's question**s**"?

Reply:
Resolved in 57dc05f2eab82b68a8799ee110cb136909050604